File: programmers/etc/level1/하샤드_수.py
def solution(x):
    return x%sum([int(i) for i in str(x)]) == 0

x = 10
print(solution(x)) # True

x = 11
print(solution(x)) # True

x = 12
print(solution(x)) # False

x = 13
print(solution(x)) # False

File: Java/programmers/etc/level1/하샤드수.java
package CodeTest.Java.programmers.etc.level1;

public class 하샤드수 {
    public boolean solution(int x) {
        int tmp = 0;

        for (String i : String.valueOf(x).split("")) {
            tmp += Integer.parseInt(i);
        }
        
        return x%tmp == 0;
    }
}
